tests/test_read_configs_include.py

```diff
@@ -12,6 +12,15 @@ def root_dir(request):
     return os.path.dirname(os.path.dirname(request.fspath))
 
 
+@pytest.fixture(scope="function")
+def run_in_compose_dir(root_dir):
+    compose_dir = os.path.join(root_dir, "birdhouse")
+    old_cwd = os.getcwd()
+    os.chdir(compose_dir)
+    yield
```

Comment:
Just so that we don't end up in an unexpected directory in case there is an error raised in some test I'd recommend changing this to:

```python
try:
    yield
finally:
    os.chdir(old_cwd)
```